src/screens/SignIn.js

```diff
@@ -149,8 +148,6 @@ export default class SignIn extends Component {
 					<Button text='Create Account' onPress={() => this._handleCreateAccount()} />
 				</View>
 
-				<KeyboardSpacer/>
-
 			</SafeAreaView>
```

Comment:
I don't think we should remove `KeyboardSpacer` from this view since now the keyboard would cover the Sign In and Create Account button on iPhone. Looks like there is a `topSpacing` prop ([documentation](https://github.com/Andr3wHur5t/react-native-keyboard-spacer)) that you might be able to play around with, but you'd have to give it a number which might not be the best solution given the various screen sizes we're working with.

I think one solution that might work well for this is to use react native's platform check to put in the KeyboardSpacer if it's iOS:

```suggestion
				{Platform.OS === 'ios'
					? <KeyboardSpacer/>
					: null}

			</SafeAreaView>
```
![simulator screen shot - iphone x - 2018-11-28 at 19 04 46](https://user-images.githubusercontent.com/5591523/49190469-8afb9480-f340-11e8-84ed-6e46f144aafa.png)

Reply:
Oh that's a good suggestion! I didn't test this in iOS at all after making the change so that's my bad. I'll put the iOS check in.